api/client/proto/authservice.proto

```diff
@@ -755,6 +755,10 @@ message DatabaseCSRRequest {
     bytes CSR = 1 [ (gogoproto.jsontag) = "csr" ];
     // ClusterName is the name of the cluster the request is for.
     string ClusterName = 2 [ (gogoproto.jsontag) = "cluster_name" ];
+    // SignWithDatabaseCA if set to true will use Database CA to sign the created certificate.
```

Comment:
Could you add the comment `DELETE IN teleport version`  indicating that this filed is temporary and can be deleted in next releases taking in account [compatibility promise  ](https://github.com/gravitational/teleport/blob/master/rfd/0012-teleport-versioning.md)